src/static/js/pad.js

```diff
@@ -554,10 +554,10 @@ var pad = {
         pad.changeViewOption('rtlIsTrue', true);
       }
 
-      var fonts = ['useMonospaceFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
+
+      var fonts = ['useMonospaceFont', 'useMontserratFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
         'useLucidaFont', 'useLucidaSansFont', 'usePalatinoFont', 'useRobotoMonoFont', 'useTahomaFont', 'useTimesNewRomanFont',
-        'useTrebuchetFont', 'useVerdanaFont', 'useSymbolFont', 'useWebdingsFont', 'useWingDingsFont', 'useSansSerifFont',
-        'useSerifFont'];
```

Comment:
Oh it seems like it, sorry. I was merging the lines were the two fonts were added together and I guess those 2 lines kinda slipped. Very sorry about that.

Gonna add them again (hope there's some --amend functionality here on github ), as soon as I can, I'm in class right now and don't have my laptop with me.